config/mogoose.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const MONGO_URL = `mongodb+srv://budgetify-api:${process.env.MONGO_KEY}@budgetifycluster.7xfxj.mongodb.net/budgetify?retryWrites=true&w=majority`;
```

Comment:
You can store the whole mongo connection string in ENV.

Reply:
Ok, I did